src/Interfaces/AbstractPolytope.jl

```diff
@@ -96,6 +97,60 @@ function isempty(P::AbstractPolytope)::Bool
     return isempty(vertices_list(P))
 end
 
+"""
+    chebyshev_center(P::AbstractPolytope{N};
+                     backend=default_polyhedra_backend(P, N),
+                     compute_radius::Bool=false) where {N<:Real}
+
+Compute the [Chebyshev center](https://en.wikipedia.org/wiki/Chebyshev_center)
+of a polytope.
+
+### Input
+
+- `P`              -- polytope
+- `backend`        -- (optional, default: `default_polyhedra_backend(P, N)`) the
+                      backend for polyhedral computations
+- `compute_radius` -- (optional; default: `false`) option to additionally return
+                      the radius of the largest ball enclosed by `P` around the
+                      Chebyshev center
+
+### Output
+
+If `compute_radius` is `false`, the result is the Chebyshev center of `P`.
+If `compute_radius` is `true`, the result is the pair `(c, r)` where `c` is the
+Chebyshev center of `P` and `r` is the radius of the largest ball with center
+`c` enclosed by `P`.
+
+### Notes
+
+The Chebyshev center is the center of a largest Euclidean ball enclosed by `P`.
+In general, the center of such a ball is not unique (but the radius is).
+
+Formally, letting ``∂P`` denote the boundary of ``P``, the Chebyshev center is
+
+```math
+    \\arg\\min_c \\min_{x ∈ ∂P} ‖x - c‖₂.
+```
+"""
+function chebyshev_center(P::AbstractPolytope{N};
+                          backend=default_polyhedra_backend(P, N),
+                          compute_radius::Bool=false
+                         ) where {N<:Real}
+    require(:Polyhedra; fun_name="chebyshev_center")
+    require(:JuMP; fun_name="chebyshev_center")
+    require(:GLPK; fun_name="chebyshev_center")
+    @assert applicable(polyhedron, P) "try converting the set to HPolytope " *
```

Comment:
this seems like an unnecessary / implementation detail restriction. you can assert that the set is polyhedral, such as checking the `constraints_list` is applicable, then let `P = convert(HPolytope, P)` and pass it to Polyhedra's `chebyshevcenter`. a specialized version if the vrep is available is anyways not implemented yet (see [here](https://github.com/JuliaPolyhedra/Polyhedra.jl/blob/737e49135c5bbfcd1e754398ec449d57e05aec2f/src/center.jl#L59))

Reply:
This is what I tried to express in https://github.com/JuliaReach/LazySets.jl/pull/1696#discussion_r324466196.

Reply:
Ok. It is not strictly needed as one can first convert `P` to HPolyhedron and then apply `LazySets.polyhedron`. This was the original idea of `polyhedron`, to be a lower-level conversion from a lazy polyhedron to a concrete one from Polyhedra.

Reply:
Yes, but it is a bit annoying. In particular, if your set prefers V-rep, this conversion would be bad. But as you pointed out, the V-rep is currently not supported by `Polyhedra`, so for now this seems valid.

Reply:
As such, imo it's not particularly interesting to add conversions from different lazysets types to Polyhedra (https://github.com/JuliaReach/LazySets.jl/issues/1505), since they anyways have to pass through HPolytope or HPolyhedron (unless we also convert rays, lines, halfspaces directly -- but again i see no interest in doing so ATM).

Reply:
> As such, imo it's not particularly interesting to add conversions from different lazysets types to Polyhedra (#1505), since they anyways have to pass through HPolytope or HPolyhedron

This is specific to this function, right? In general it can be helpful.